ime/app/src/main/java/com/anysoftkeyboard/keyboards/KeyboardSwitcher.java

```diff
@@ -294,7 +294,13 @@ private AnyKeyboard getSymbolsKeyboard(int keyboardIndex) {
         ensureKeyboardsAreBuilt();
         AnyKeyboard keyboard = mSymbolsKeyboardsArray[keyboardIndex];
 
-        if (keyboard == null || keyboard.getKeyboardMode() != mKeyboardRowMode) {
+        // if it's one of mirfatif keyboards
+        boolean mirfatifKeyboard = mirfatifKeyboard();
```

Comment:
what is this type of keyboard?

Reply:
One of the two keyboards added by me in other PRs: [modified QWERTY Latin keyboard](https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2409) and [Urdu keyboard](https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2408).

The changes I made in this PR affect only the two keyboards I'm adding. All other keyboards will keep on functioning as they were.